sky/users/permission.py

```diff
@@ -21,6 +22,17 @@
 logging.getLogger('casbin.role').setLevel(sky_logging.ERROR)
 logging.getLogger('casbin.model').setLevel(sky_logging.ERROR)
 logging.getLogger('casbin.rbac').setLevel(sky_logging.ERROR)
+
+# Suppress SQLAlchemy warnings about duplicate class names in
+# sqlalchemy_adapter
```

Comment:
Why does it raise this warning at the first place?

Reply:
sqlalchemy_adapter v1.4.0 introduced a change in https://github.com/officialpycasbin/sqlalchemy-adapter/pull/4, where if no `data_class` is set when initialize `Adapter`, it will create a [new data class](https://github.com/officialpycasbin/sqlalchemy-adapter/blob/v1.6.0/sqlalchemy_adapter/adapter.py#L57) with the same name `CasbinRule`, which is same with the [default one](https://github.com/officialpycasbin/sqlalchemy-adapter/blob/v1.6.0/sqlalchemy_adapter/adapter.py#L61). 
I updated the PR to use the default `CasbinRule` when initialize `Adapter`.